Sources/Xcodeproj/Module+PBXProj.swift

```diff
@@ -298,3 +291,25 @@ extension Module  {
         return buildSettings
     }
 }
+
+extension Plist {
+    // FIXME: This is only internal for unit testing.
+    /// Quotes the strings in the Plist.
+    func quoteStrings() -> Plist {
+        switch self {
+        case .string(let str):
+            // Only quote if string has a space character.
+            guard str.utf8.contains(UInt8(ascii: " ")) else {
```

Comment:
Always quoting rules specific to each build setting? because quoting some settings became invalid for eg Product Module became `"Hello"`